eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java

```diff
@@ -49,13 +70,32 @@
     requiresDependencyResolution = ResolutionScope.COMPILE
 )
 @SuppressWarnings("PMD.LongVariable")
-public final class BinarizeMojo extends SafeMojo {
+public final class BinarizeMojo extends SafeMojo implements CompilationStep {
 
     /**
      * The directory where to binarize to.
      */
     public static final String DIR = "binarize";
 
+    /**
+     * Parsing train with XSLs.
+     */
+    static final Train<Shift> TRAIN = new TrBulk<>(
+        new TrClasspath<>(
+            new ParsingTrain()
+                .empty()
+                .with(new StUnhex())
+        ),
+        Arrays.asList(
+            "/org/eolang/maven/add_rust/add_rust.xsl"
+        )
+    ).back().back();
+
+    /**
+     * Extension for compiled sources in XMIR format (XML).
+     */
+    static final String EXT = "xmir";
```

Comment:
@levBagryansky Why this field is not private?

Reply:
this field was created for the future